dependencies/requirements/requirements.txt

```diff
@@ -12,7 +12,7 @@ google-cloud-monitoring
 grain[parquet]
 huggingface_hub
 jax!=0.7.1
-jaxlib!=0.7.1
+jaxlib==0.8.0
```

Comment:
Thanks for the change! I think we could follow this guide https://github.com/AI-Hypercomputer/maxtext/blob/main/docs/guides/update_dependencies.md to update GPU dependency only with seed-env (base_requirements/gpu-base-requirements.txt)?